pkg/network/network-policy.go

```diff
@@ -0,0 +1,33 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+)
+
+func RenderNetworkPolicy(manifestDir string, info ClusterInfo) ([]*unstructured.Unstructured, error) {
+	data := render.MakeRenderData()
+	data.Data["Namespace"] = os.Getenv("OPERAND_NAMESPACE")
+	if info.OpenShift4 {
+		data.Data["ClusterDNSNamespace"] = "openshift-dns"
```

Comment:
this wont work on OKD
kubevirt-hyperconverged is the ns there

i am also not sure we are allowed to mention this ns here

what about using the ns where CNAO is deployed instead line 17 (under same condition)

Reply:
ye corrected myself

anyhow it might be fine in that aspect
https://docs.okd.io/latest/networking/networking_operators/dns-operator.html